README.md

```diff
@@ -1,23 +1,20 @@
 # ValaVerbalExpressions
 Regular expressions in Vala made easy
 
-
 ## Installing / Getting started
 
-To build this library, you need [Stew](https://github.com/desiderantes/stew), and Vala >= 0.28
+To build this library, you need [Meson](https://mesonbuild.com), and Vala >= 0.28
 
 ```shell
 git clone https://github.com/VerbalExpressions/ValaVerbalExpressions.git
 cd ValaVerbalExpressions/
-stew install
+mkdir build
+cd build
+meson ..
+ninja build
```

Comment:
It is just `ninja` on its own, or `ninja src/libverbex-1.0.so` if you'd like to build a particular target.